src/build_workflow/build_args.py

```diff
@@ -95,6 +96,13 @@ def __init__(self) -> None:
             default="tar",
             dest="distribution"
         )
+        parser.add_argument(
+            "--continue-on-error",
+            dest="continue_on_error",
+            default=False,
+            action="store_true",
+            help="Do not fail the distribution build on any plugin component failure.",
```

Comment:
```suggestion
            help="Do not fail the distribution build on any component failure.",
```

Reply:
any component would include OpenSearch and OSD as well which is not the case. We need to fail the build the if core component fails.